### PR TITLE
Css snapshot module

### PR DESCRIPTION
**Pull request by enrrou**
Resolves https://github.com/opentelekomcloud/ansible-collection-cloud/issues/143

### CODE REVIEW

**Comment by enrrou** [comment]
recheck

File: meta/runtime.yml [hunk 1]
```diff
@@ -21,7 +21,9 @@ action_groups:
     - cce_node_pool
     - css_cluster
     - css_cluster_info
+    - css_snapshot
     - css_snapshot_info
+    - css_snapshot
```
Comment (lego963) [inline meta/runtime.yml]: repeating

File: plugins/modules/css_snapshot.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+module: css_snapshot
+short_description: Manage CSS snapshots
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.9.0"
```
Comment (lego963) [inline plugins/modules/css_snapshot.py:18]: update version please

File: tests/sanity/ignore-2.9.txt [hunk 3]
```diff
@@ -23,6 +23,8 @@ plugins/modules/ces_quotas_info.py validate-modules:missing-gplv3-license
 plugins/modules/cce_cluster_node_info.py validate-modules:missing-gplv3-license
 plugins/modules/css_cluster.py validate-modules:missing-gplv3-license
 plugins/modules/css_cluster_info.py validate-modules:missing-gplv3-license
+plugins/modules/css_cluster_info.py validate-modules:missing-gplv3-license
```
Comment (anton-sidelnikov) [inline tests/sanity/ignore-2.9.txt:26]: duplicated entry

File: plugins/modules/css_snapshot.py (hunk 2 above)
Comment (anton-sidelnikov) [inline plugins/modules/css_snapshot.py:18]: 0.10.0?

File: tests/sanity/ignore-2.9.txt [hunk 4]
```diff
@@ -23,6 +23,8 @@ plugins/modules/ces_quotas_info.py validate-modules:missing-gplv3-license
...
+plugins/modules/css_snapshot.py validate-modules:missing-gplv3-license
 plugins/modules/css_snapshot_info.py validate-modules:missing-gplv3-license
```
Comment (lego963) [inline tests/sanity/ignore-2.9.txt:28]: repeating

**Review by lego963 — CHANGES_REQUESTED** [review]

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Pull request merged**